tests/test_unit/test_cdf_tk/test_commands/test_purge.py

```diff
@@ -0,0 +1,201 @@
+                        )
+                    },
+                    1,
+                    1,
+                    None,
+                    None,
+                    None,
+                    None,
+                    True,
+                    "node",
+                    is_global=True,
+                )
+            ]
+        )
+        response = MagicMock(spec=Response)
+        response.json.return_value = {"items": [ts.dump() for ts in cognite_timeseries_2000_list]}
+        client.post.return_value = response
+        client.data_modeling.instances.aggregate.return_value = CountValue(
+            "externalId", len(cognite_timeseries_2000_list)
+        )
+        client.time_series.retrieve_multiple.side_effect = retrieve_timeseries_mock
+        client.time_series.unlink_instance_ids.side_effect = unlink_instance_ids_mock
+        yield client
+
+
+class TestPurgeInstances:
+    def test_purge_dry_run_unlink(self, purge_client: ToolkitClient) -> None:
+        cmd = PurgeCommand(silent=True)
+        cmd.instances(
+            client=purge_client,
+            view=["cdf_cdm", "CogniteTimeSeries", "v1"],
+            instance_space=None,
+            dry_run=True,
+            auto_yes=True,
+            unlink=True,
+        )
+
+        assert purge_client.time_series.retrieve_multiple.call_count == 2
+        assert purge_client.data_modeling.instances.delete_fast.call_count == 0
+
+    def test_purge_dry_run(self, purge_client: ToolkitClient) -> None:
+        cmd = PurgeCommand(silent=True)
+        cmd.instances(
+            client=purge_client,
+            view=["cdf_cdm", "CogniteTimeSeries", "v1"],
+            instance_space=None,
+            dry_run=True,
+            auto_yes=True,
+            unlink=False,
+        )
+
+        assert purge_client.time_series.unlink_instance_ids.call_count == 0
+        assert purge_client.data_modeling.instances.delete_fast.call_count == 0
+
+    def test_purge_unlink(self, purge_client: ToolkitClient) -> None:
+        cmd = PurgeCommand(silent=True)
+        cmd.instances(
+            client=purge_client,
+            view=["cdf_cdm", "CogniteTimeSeries", "v1"],
+            instance_space=None,
+            dry_run=False,
+            auto_yes=True,
+            unlink=True,
+        )
+
+        assert purge_client.time_series.unlink_instance_ids.call_count == 2
+        assert purge_client.data_modeling.instances.delete_fast.call_count == 2
+
+    def test_purge(self, purge_client: ToolkitClient) -> None:
+        cmd = PurgeCommand(silent=True)
+        cmd.instances(
+            client=purge_client,
+            view=["cdf_cdm", "CogniteTimeSeries", "v1"],
+            instance_space=None,
+            dry_run=False,
+            auto_yes=True,
+            unlink=False,
+        )
+
+        assert purge_client.time_series.unlink_instance_ids.call_count == 0
+        assert purge_client.data_modeling.instances.delete_fast.call_count == 2
```

Comment:
These are a _little_ thin. Do you think you could check that the methods are called with the correct filters/values at least?